### PR TITLE
add 'displayName' field to Asset type

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- add `displayName` to types: `Asset`, `ClusterAsset`, `Source`
- rename one test
- ...

**Related issue(s)**

https://github.com/kyma-project/rafter/issues/71
https://github.com/kyma-project/console/issues/1617

### CODE REVIEW

**Comment by kazydek** [comment]
Could you also update [Rafter docs](https://kyma-project.io/docs/master/components/rafter/#custom-resource-custom-resource)? We describe all Rafter custom resource definitions.

File: pkg/apis/rafter/v1beta1/asset_types.go [hunk 1]
```diff
@@ -28,8 +28,9 @@ type Asset struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   AssetSpec   `json:"spec,omitempty"`
-	Status AssetStatus `json:"status,omitempty"`
+	Spec        AssetSpec   `json:"spec,omitempty"`
+	Status      AssetStatus `json:"status,omitempty"`
+	DisplayName string      `json:"displayName,omitempty"`
```
Comment (aerfio) [inline pkg/apis/rafter/v1beta1/asset_types.go]: Please add DisplayName inside CommonAssetSpec field

Comment (michal-hudy) [inline pkg/apis/rafter/v1beta1/asset_types.go]: It shouldn't be added here. It should be added to `CommonAssetSpec` as @aerfio mentioned.

File: pkg/apis/rafter/v1beta1/asset_common_types.go [hunk 2]
```diff
@@ -37,8 +37,9 @@ type AssetStatusRef struct {
 }
 
 type AssetFile struct {
-	Name     string                `json:"name"`
-	Metadata *runtime.RawExtension `json:"metadata,omitempty"`
+	Name        string                `json:"name"`
+	DisplayName string                `json:"displayName,omitempty"`
```
Comment (michal-hudy) [inline pkg/apis/rafter/v1beta1/asset_common_types.go]: `AssetFile` is a file representation in `Status`. It is a `DisplayName` for Asset, not for a file, so it should be added to the `CommonAssetSpec`

File: internal/controllers/clusterassetgroup_controller_test.go [hunk 3]
```diff
@@ -49,7 +49,7 @@ var _ = Describe("Asset", func() {
 		}
 	})
 
-	It("should successfully create, update and delete AssetGroup", func() {
+	It("should successfully create, update and delete ClusterAssetGroup", func() {
```
Comment (michal-hudy) [inline internal/controllers/clusterassetgroup_controller_test.go:52]: Please update this test to verify if `displayName` has been properly propagated from `ClusterAssetGroup` to `ClusterAsset`. Same for `AssetGroups`

**Pull request merged**